docs/configuration/configuration.md

```diff
@@ -330,6 +346,10 @@ Consul setups, the relevant address is in `__meta_consul_service_address`.
 In those cases, you can use the [relabel](#relabel_config)
 feature to replace the special `__address__` label.
 
+The Consul API only has basic support for filtering nodes (currently by node
```

Comment:
This should be worded so that relabelling is the primary way to do things, and the other options only exist for performance reasons. I want to avoid users trying to add features just because they don't like relabelling.

Reply:
should be a bit better now, I'm open to suggestion if not clear enough